ppml/trusted-bigdata/settings.xml

```diff
@@ -0,0 +1,37 @@
+    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
why need maven settings? this can not be committed

Reply:
This is to speed up mvn, it will be deleted before merge